traducidos/SUMMARY.md

```diff
@@ -0,0 +1,6 @@
+# Resumen
+
+* [Introducción](README.md)
+* [Información Básica](intro/README.md)
+* [Reglas del formador, consejos prácticos](tips/README.md)
```

Comment:
En español de España, *coach* se podría traducir como *formador*. Si en español de latinoamérica tenéis una palabra mejor, cambiadlo.